### PR TITLE
docs(readme): add documentation regarding salt.formula and salt.master, addressing #326

### PR DESCRIPTION
**Pull request by droope**
### PR progress checklist (to be filled in by reviewers)

- [x] Changes to documentation are appropriate (or tick if not required)
- [x] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [ ] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [x] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Related issues and/or pull requests

This addresses #326, small doc change.

### Describe the changes you're proposing

Some small documentation to make it easier for new users.

### Pillar / config required to test the proposed changes

n/a

### Debug log showing how the proposed changes work

n/a

### Documentation checklist

- [ x] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [ ] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

**Comment by dafyddj** [comment]
Hi, and thanks for your contribution. Could you shorten the commit message title to 72 chars as noted by the failing `commitlint` check?

File: docs/README.rst [hunk 1]
```diff
@@ -126,12 +126,17 @@ Undo the effects of ``salt.pkgrepo`` on Debian, RedHat, and SuSE.
 
 Clone selected `Salt formulas
 <http://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html>`_
-Git repositories under ``/srv/formulas`` and makes them available in the
-relevant ``file_roots`` settings. Pillar data can be used to customize all
-paths, URLs, etc.
+Git repositories under ``/srv/formulas`` and makes them available in the relevant ``file_roots`` settings. Please note that in order for `file_roots` to be updated, `salt.master` must be called after `salt.formulas`. For example:
```
Comment (myii) [inline docs/README.rst:129]: Need to use double-backticks in reStructuredText:

```suggestion
Git repositories under ``/srv/formulas`` and makes them available in the relevant ``file_roots`` settings. Please note that in order for ``file_roots`` to be updated, ``salt.master`` must be called after ``salt.formulas``. For example:
```

@dafyddj I'm sure one of our linters was supposed to catch this but that doesn't seem to be the case.  Any ideas?

**Review by myii — CHANGES_REQUESTED** [review]
Please fix the formatting as suggested.

**Comment by droope** [comment]
Hi.

I made the changes you requested:

* Shortened the commit message.
* Used double backticks instead of single backticks.

**Comment by droope** [comment]
Oh thank you! awesome 👍🏼

**Pull request merged**